include/revng-c/mlir/Dialect/Clift/Utils/Helpers.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <optional>
+
+#include "mlir/IR/FunctionInterfaces.h"
+
+#include "revng/Support/IRHelpers.h"
+#include "revng/Support/MetaAddress.h"
+
+namespace mlir::clift {
+
+inline std::optional<MetaAddress> getMetaAddress(mlir::FunctionOpInterface F) {
```

Comment:
Usually we don't use `std::optional<MetaAddress>`, `MetaAddress` already embeds an invalid encoding (i.e., `MetaAddress::invalid()`. Unless you have a really good reason, use `MetaAddress::invalid` instead of `std::nullopt`.

Reply:
Done.